### ISSUE TITLE
SF Symbol name for device

### ISSUE THREAD

**Issue by matiaskorhonen** [devicekit/DeviceKit#280]
It would be great if DeviceKit could return the name of the correct SF symbol for a device.

For example:

```swift
Device.current.symbol_name

// or

Device.current.system_image
```

So, an iPhone 8 or earlier would return `iphone.homebutton` and home buttonless phones would return `iphone` and so forth…

| `iphone`  | `iphone.homebutton` | `ipad`  | `ipad.homebutton`  |
|-|-|-|-|
| <img width="124" alt="iphone" src="https://user-images.githubusercontent.com/43314/116360575-bbdd9d80-a808-11eb-89cf-96609d07d259.png"> | <img width="123" alt="iphone.homebutton" src="https://user-images.githubusercontent.com/43314/116360618-c9932300-a808-11eb-83de-3bf88fa84cc1.png"> | <img width="126" alt="ipad" src="https://user-images.githubusercontent.com/43314/116360730-e16aa700-a808-11eb-98f0-055dd923628e.png"> | <img width="123" alt="ipad.homebutton" src="https://user-images.githubusercontent.com/43314/116360792-f1828680-a808-11eb-8932-2d5d8d039b7d.png"> |

**Comment by denisenepraunig** [comment]
Hahah, that would be a nice feature 😅. @Zandor300 what do you think?

**Comment by Zandor300** [comment]
@denisenepraunig 
Yeah, really nice. Could be implemented fairly easily I think using just Swift:

```swift
var sfSymbolName: String {
    if isPhone {
        return hasRoundedDisplayCorners ? "iphone" : "iphone.homebutton"
    } else {
        return hasRoundedDisplayCorners ? "ipad" : "ipad.homebutton"
    }
}
```

**Comment by gabors** [comment]
Great idea, let's not forget Macs and Apple TVs

**Comment by kudit** [comment]
This feature has been added in this project: GitHub.com/kudit/Device
It includes support for all 3 "generations" of iPhone (home button, notch, & Dynamic Island), 2 models of iPad, HomePod, Apple TV, Apple Vision, iPod touch, and Macs! (I could use some help adding more Mac models but there is an easy way to add additional devices without having to learn an obscure mapping).
